core/templates/dev/head/components/RuleTypeSelectorDirective.js

```diff
@@ -80,15 +74,16 @@ oppia.directive('ruleTypeSelector', [function() {
           }
         });
 
+        // Select the first choice by default.
+        if (!$scope.localValue) {
+          $scope.localValue = choices[0].id;
+          $scope.onSelectionChange()($scope.localValue);
+        }
+
         // Initialize the dropdown.
         $(select2Node).select2('val', $scope.localValue);
 
-        // Update $scope.localValue when the selection changes.
         $(select2Node).on('change', function(e) {
-          $scope.localValue = e.val;
-          // This is needed to actually update the localValue in the containing
-          // scope.
-          $scope.$apply();
           $scope.onSelectionChange()(e.val);
           // This is needed to propagate the change and display input fields for
           // parameterizing the rule.
```

Comment:
I think, we can delete the line below, i.e $scope.apply(). as $scope.onSelectionChange()(e.val) is in angular scope. 
Do you know how to reproduces what this comment says?